packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/tera-calibrate-ensemble-ciemss-drilldown.vue

```diff
@@ -190,50 +190,75 @@
-					<tera-simulate-chart
-						v-for="(cfg, index) of node.state.chartConfigs"
-						:key="index"
-						:run-results="runResults"
-						:chartConfig="{
-							selectedRun: props.node.state.postForecastId,
-							selectedVariable: cfg
-						}"
-						has-mean-line
-						@configuration-change="chartProxy.configurationChange(index, $event)"
-						@remove="chartProxy.removeChart(index)"
-						show-remove-button
-						:size="chartSize"
-						class="mb-2"
-					/>
-					<Button
-						class="p-button-sm p-button-text"
-						@click="chartProxy.addChart()"
-						label="Add chart"
-						icon="pi pi-plus"
-					/>
+				<section class="pb-3" ref="outputPanel">
+					<Accordion multiple :active-index="[0, 1]" class="px-2">
+						<!-- <AccordionTab header="Summary">
+						</AccordionTab> -->
+						<AccordionTab header="Loss">
+							<vega-chart
+								v-if="!_.isEmpty(lossValues)"
+								expandable
+								ref="lossChartRef"
+								:are-embed-actions-visible="true"
+								:visualization-spec="lossChartSpec"
+							/>
+						</AccordionTab>
+						<template v-if="!isRunInProgress">
+							<AccordionTab header="Ensemble variables over time">
+								<template v-for="setting of selectedEnsembleVariableSettings" :key="setting.id">
+									<vega-chart
+										expandable
+										:are-embed-actions-visible="true"
+										:visualization-spec="ensembleVariableCharts[setting.id]"
+									/>
+								</template>
+							</AccordionTab>
+						</template>
+					</Accordion>
+					<tera-progress-spinner v-if="isRunInProgress" :font-size="2" is-centered style="height: 100%">
+						{{ node.state.currentProgress }}%
+					</tera-progress-spinner>
 				</section>
-
-				<tera-progress-spinner
-					v-if="inProgressCalibrationId || inProgressForecastId"
-					:font-size="2"
-					is-centered
-					style="height: 100%"
-				>
-					{{ node.state.currentProgress }}%
-				</tera-progress-spinner>
 			</tera-drilldown-section>
 		</template>
+		<template #sidebar-right>
+			<tera-slider-panel
+				v-model:is-open="isOutputSettingsPanelOpen"
+				direction="right"
+				class="input-config"
+				header="Output settings"
+				content-width="360px"
+			>
+				<template #overlay>
+					<tera-chart-settings-panel
+						:annotations="
+							[ChartSettingType.VARIABLE_ENSEMBLE].includes(activeChartSettings?.type as ChartSettingType)
+								? getChartAnnotationsByChartId(activeChartSettings?.id ?? '')
+								: undefined
+						"
+						:active-settings="activeChartSettings"
+						:generate-annotation="generateAnnotation"
+						@delete-annotation="deleteAnnotation"
+						@close="activeChartSettings = null"
+					/>
+				</template>
+				<template #content>
+					<div class="output-settings-panel">
+						<tera-chart-settings
+							:title="'Ensemble variables over time'"
+							:settings="chartSettings"
+							:type="ChartSettingType.VARIABLE_ENSEMBLE"
+							:select-options="ensembleVariables"
```

Comment:
I suspect we do not want to disallow the user from selecting regular model variables
![image](https://github.com/user-attachments/assets/97b3b2d8-049c-4ee8-8e03-2cbd5c0413b9)

For example I do not see a reason to limit the user to `Sus` and disallow `Susceptible_state`, `S`, and 'Susceptible'. 
If this is a more complex change than I am aware it may be worth doing in a follow up PR I guess